cozytouch.py

```diff
@@ -1076,16 +1076,12 @@ def gestion_consigne(texte,url_device,nom_device, idx_cons_domoticz, cons_device
     if e != 0.5: 
         cons_domoticz = round(cons_domoticz)
 
-    # si c'est une consigne de dérogation, elle est initialisée / peut être remise à 0 par Cozytouch si changement sur le radiateur en local ou via l'application,
-    # ou si on met la consigne "eco" ou "confort".
-    # on force la consigne de dérogation précédente à 1 pour voir le changement
-    # la consigne de dérogation est ensuite traitée comme n'importe quel type de consigne
+    # si c'est une consigne de dérogation, elle est initialisée / peut être remise à 0 par Cozytouch si changement sur le radiateur en local
+    #  ou via l'application, ou si on met la consigne "eco" ou "confort".
     # TO DO : ne pas tester le texte passé en argument de la fonction mais l'objet
-    if texte=="derogation" and cons_domoticz_prec == 0:
-        cons_domoticz_prec = 1
 
     # comparaison avec la consigne en cours
-    if cons_device != cons_domoticz and cons_domoticz != cons_domoticz_prec :
+    if cons_device != cons_domoticz and cons_domoticz != cons_domoticz_prec and cons_domoticz_prec > 0 and (texte != "derogation" or cons_domoticz > 0):
```

Comment:
Lorsque la consigne de dérogation est à 0, et que l'on positionne via domoticz une temperature comprise entre eco et confort, alors nous ne passons pas dans cette condition puisque la con_domoticz_prec vaut 0.
Je propose de remplacer par 
_if cons_device != cons_domoticz and cons_domoticz != cons_domoticz_prec and (cons_domoticz > 0 or texte == "derogation"):_
Mais on risque peut être de retomber dans ton cas d'erreur ?